packages/botonic-core/src/plugins.js

```diff
@@ -23,19 +23,22 @@ export async function runPlugins(
   session,
   lastRoutePath,
   response = null,
-  parsedResponse = null
+  parsedResponse = null,
+  dataProvider = null
 ) {
   for (const key in plugins) {
     const p = await plugins[key]
     try {
-      if (mode == 'pre') await p.pre({ input, session, lastRoutePath })
+      if (mode == 'pre')
```

Comment:
Preference of `===`. If `botonic-dx` doesn't have it, we should consider to add it.

Reply:
fixed!

Reply:
added the rule in `botonic-eslint-config` package: https://eslint.org/docs/rules/eqeqeq
https://github.com/hubtype/botonic/pull/1764/commits/22697215a62c626661bb50fa6b7e37eb9cfcae4d